paper/paper.md

```diff
@@ -0,0 +1,109 @@
+
+To address these challenges, we developed `REDCapTidieR` as an open source R package that transforms the standard REDCap output into a format that adheres to tidy data principles. `REDCapTidieR` has the potential to save organizations and research staff immeasurable amounts of time, allowing them to quickly query their data without the need for intricate data parsing processes.
+
+# Statement of Need
+
+As of 2023, the REDCap Consortium boasts nearly 3 million users across over 150 countries. REDCap databases exhibit significant variation in complexity, ranging from simple tables with easily identifiable records to more challenging scenarios where pinpointing a unique identifier is harder. This complexity often arises in databases that make use of "repeating instruments" and "repeating events." For an in-depth exploration of this concept, refer to the [`REDCapTidieR` documentation](https://chop-cgtinformatics.github.io/REDCapTidieR/articles/diving_deeper.html#longitudinal-redcap-projects). Fundamentally, repeating events and instruments support longitudinal studies, where subjects may have distinct timelines with varying levels of record granularity. This is where the flattening of the database into the block matrix becomes a pain point for analysts.
+
+While there are a few existing REDCap tools for R documented by [`REDCap-tools`](https://redcap-tools.github.io/projects/), `REDCapTidieR` occupies a unique space by providing analysts with an opinionated framework that quickly prepares them for data analysis. Although some of the aforementioned tools also offer functions for data processing, such as the [`tidyREDCap`](https://raymondbalise.github.io/tidyREDCap/) and [`REDCapDM`](https://ubidi.github.io/REDCapDM/index.html) packages, `REDCapTidieR` is unique in how it restructures the block matrix into a format that is easily interpretable within the user's programmatic environment. Of the tools available, `REDCapTidieR` is the only one that fundamentally restructures the block matrix in its entirety.
+
+REDCapTidieR was developed with deployment in production environments as a key consideration. To ensure the utmost confidence in the handling of user data, we've implemented an extensive test suite that exhibits 98% code coverage, as of the package's version 1.0 release. Ample documentation is accessible through a collection of package vignettes and articles, offering detailed insights into the opinionated framework, design structure, and a comprehensive glossary of terms associated with the REDCapTidieR package. These considerations have earned the package an [OpenSSF Best Practices certification](https://www.bestpractices.dev/en/projects/6845) [@openssf_cit], which certifies open source projects that meet stringent criteria for delivering high-quality and secure software.
+
+| Package     | Data Export Support | Data Import Support | Data Manipulation | Data Tidying | Production Capability |
```

Comment:
I'm not sure "Data Tidying" or "Production Capability" are well defined here.

Instead of "Data Tidying" maybe we want something that describes the results `REDCapTidier` gives you out of the box. "Analysis Ready Data"? "Produces Tidy Data"? Anyone can say their package does "data tidying" but I think only we can say it gives you "tidy data".

I'm not sure what to do with "Production Capability" to be honest. The manuscript emphasizes the test suite so I'd be cautious of giving only us and `REDCapR` a check without more investigation. In general, I think "production" is an ill-defined term that we need to define if we want to use it in a table like this.

Reply:
Yea I agree with this, I found it hard to prop up the package over other ones in a clear way. I'm not sure about "Analysis Ready Data" or "Produces Tidy Data" either because you could say a similar thing about `tidyREDCap`, but that moreso gives you a piece of your data. `REDCapTidieR` instead restructures everything... Maybe "Tidy Reformatting"?

I think the key with "production capability" or whatever it represents is that it should encompass `REDCapR` and `REDCapTidieR` as open source packages that have been vetted with stringent practices.